components/python/atomicwrites/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+#
+
+BUILD_BITS=NO_ARCH
+BUILD_STYLE=setup.py
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		atomicwrites
+COMPONENT_VERSION=	1.3.0
+COMPONENT_FMRI=		library/python/atomicwrites
+COMPONENT_CLASSIFICATION=Development/Python
+COMPONENT_SUMMARY=	atomicwrites - Python Atomic File Writes
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:75a9445bac02d8d058d5e1fe689654ba5a6556a1dfd8ce6ec55a0ed79866cfa6
+COMPONENT_ARCHIVE_URL=	$(call pypi_url)
+COMPONENT_PROJECT_URL=	https://github.com/untitaker/python-atomicwrites
+COMPONENT_LICENSE=	MIT
+COMPONENT_LICENSE_FILE=	LICENSE
+
+PYTHON_VERSIONS=$(PYTHON3_VERSIONS)
+
+include $(WS_MAKE_RULES)/common.mk
+
+COMPONENT_TEST_DIR =	$(SOURCE_DIR)/tests
+COMPONENT_TEST_CMD =	$(PYTHON) -m pytest
+COMPONENT_TEST_ARGS =
```

Comment:
We don't have pytest-37 and -39 yet. So gmake test fails
/usr/bin/python3.7 -m pytest )
/usr/bin/python3.7: No module named pytest
gmake: *** [/oibuild/oi-userland/make-rules/setup.py.mk:155: /oibuild/oi-userland/components/python/atomicwrites/build/i386-3.7/.tested] Error 1	/usr/bin/python3.7 -m pytest )
/usr/bin/python3.7: No module named pytest
gmake: *** [/oibuild/oi-userland/make-rules/setup.py.mk:155: /oibuild/oi-userland/components/python/atomicwrites/build/i386-3.7/.tested] Error 1